website/screens/components/resultset-table/code/ResultsetTableCodePage.tsx

```diff
@@ -160,6 +184,12 @@ const sections = [
         title: "Basic usage",
         content: <Example example={basicUsage} defaultIsVisible />,
       },
+      {
+        title: "Reduced usage",
+        content: (
+          <p>This example is not available yet, it will be added soon.</p>
+        ),
+      },
```

Comment:
I am not sure if this is suitable for the examples or must be only placed in 'Usage'